### ISSUE TITLE
Can't change the theme

### ISSUE THREAD

**Issue by ignxcy** [oh-my-fish/oh-my-fish#935]
Describe the problem you are having and how to reproduce it. Include any other additional context that can help us troubleshoot.

**Environment report**

```
Oh My Fish version:   7
OS type:              Linux
Fish version:         fish, version 3.6.1
Git version:          git version 2.40.1
Git core.autocrlf:    no
Warning: fish_prompt.fish is overridden.
  fish_config command persists the prompt to ~/.config/fish/functions/fish_prompt.fish
  That file takes precedence over Oh My Fish's themes. Remove the file to fix it:
  rm ~/.config/fish/functions/fish_prompt.fish

Checking for a sane environment...
If everything you use Oh My Fish for is working fine, please don't worry and just ignore the warnings. Thanks!
```
But I can't change the theme:
```
[I] $ omf theme gentoo                                                                                                          [~]
Conflicting prompt setting.
Run omf doctor and fix issues before continuing.
Theme not installed!
Install it using omf install gentoo
[I] $ omf install gentoo                                                                                                        [~]
Updating https://github.com/oh-my-fish/packages-main master... Done!
Error: gentoo already installed.
Conflicting prompt setting.
Run omf doctor and fix issues before continuing.
[I] $
```

**Comment by josealanyamatrix** [comment]
Do you know how to remove "[I] " in the prompt?

**Comment by ignxcy** [comment]
I already got it working

**Comment by soriyath** [comment]
@josealanyamatrix see https://stackoverflow.com/questions/28444740/how-to-use-vi-mode-in-fish-shell#answer-28445450
TL;DR run `fish_default_key_bindings`
